stubs/docopt/docopt.pyi

```diff
@@ -0,0 +1,11 @@
+from typing import Any, Iterable, Optional
+
+__version__: str
+
+def docopt(
+    doc: str,
+    argv: Optional[Iterable[str]] = ...,
```

Comment:
Maybe this could be made slightly more accepting (re: the split() method mentioned in the Commit message) if this was `Optional[Union[Iterable[str], str]]`?

Reply:
Yes, that would be better. I suggested a type equivalent to that earlier.

Reply:
`str` is an `Iterable[str]`

Reply:
While `Optional[Iterable[str]]` is technically correct, it doesn't make it obvious that the `hasattr(thing, 'split')` was considered when writing the stubs. That's why I preferred to be explicit.

In some cases, mypy specifically disallows relying on that (`a, b = some_string` doesn't work, for example), but I'm not sure if that applies here.

Reply:
@JelleZijlstra indeed that's true, but do you think that maybe is a bit too clever? I think listing them out separately might be more intuitive to people, e.g. `["--time", "5"]` as well as `"--time 5"` both work?